config/management/src/verify.rs

```diff
@@ -74,17 +91,36 @@ impl Verify {
             libra_global_constants::WAYPOINT,
         );
 
-        writeln!(buffer, "=================================================").unwrap();
+        write_break(&mut buffer);
+
+        if let Some(genesis_path) = self.genesis_path.as_ref() {
+            compare_genesis(storage.as_ref(), &mut buffer, genesis_path)?;
+        }
 
         Ok(buffer)
     }
 }
 
-fn write_key(storage: &dyn Storage, buffer: &mut String, key: &str) {
-    let value = storage
-        .get_public_key(key)
-        .map(|c| c.public_key.to_string())
-        .unwrap_or_else(|e| format!("{:?}", e));
+fn write_assert(buffer: &mut String, name: &str, value: bool) {
+    let value = if value { "match" } else { "MISMATCH" };
```

Comment:
nit: caps vs. no caps? Although, it seems like this probably more an intentional decision to highlight the error?

Some future food for thought: if we do end up comparing/verifying a lot of information, it might make sense to print some final result that is an aggregation of all assertions. Especially if we won't throw an error (which is really obvious to see).. this will make it less likely for human error when someone is trying to make sure state is as it should be (e.g., they don't miss a MISMATCH).

Reply:
Intentional as you can imagine. Yeah, this is a fair point, as we get feedback, I'm definitely going to iterate. It might also be worth pulling in termcolor or what not to paint green versus red.